ISISScientificComputing/autoreduce-frontend
# 1395 buttons for no variables rendered ar 1395

### Summary of work
Display re-run and back buttons for runs that don't have reduce_vars.py or run variables.

### How to test your work
Navigate to an instrument's run summary that didn't previously have the re-run or back buttons displayed. Click on the buttons to see if they function as intended.

Fixes AR-1395
